src/Invariants/Equality.jl

```diff
@@ -0,0 +1,43 @@
+using Graphs
+
+"""
+equals method for undirected edges::Union{Vector{Tuple{Int, Int}}, Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}}
+"""
+function equals(
+    a::Union{Vector{Tuple{Int, Int}}, Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}},
+    b::Union{Vector{Tuple{Int, Int}}, Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}}
+)
+
+    # convert a to an array of tuples
+    if typeof(a) == Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}
+        a = [(min(e.src, e.dst), max(e.src, e.dst)) for e in a]
+    else
+        a = [(min(e[1], e[2]), max(e[1], e[2])) for e in a]
+    end
+
+    # convert b to an array of tuples
+    if typeof(b) == Vector{Graphs.SimpleGraphs.SimpleEdge{Int64}}
+        b = [(min(e.src, e.dst), max(e.src, e.dst)) for e in b]
+    else
+        b = [(min(e[1], e[2]), max(e[1], e[2])) for e in b]
+    end
+
+    # check if the lengths are equal
+    if length(a) != length(b)
+        return false
+    end
+
+    # check if the edges are equal
+    for e in a
+        if !(e in b)
```

Comment:
Here again:

```julia 
e ∉ b && return false 
```

Reply:
BTW `∉` can be made by typing \notin + TAB

Reply:
Same question as above

Reply:
Why not replace checking the `length` and the element wise containment with :

```julia
a != b && return false 
return true 
```

Reply:
Since order does not matter. I think I will change it to use sets so we do not have to deal with this.

Reply:
Is the suggestion above more in line with Julia's style than simply `return a==b` ?

Reply:
haha silly me..... it should be `return a == b`